ansible_collections/arista/avd/roles/eos_designs/templates/designs/l3ls-evpn/interface-descriptions/underlay/ethernet-interfaces.j2

```diff
@@ -0,0 +1,5 @@
+{%- if link.type is arista.avd.defined('underlay_p2p') -%}
+P2P_LINK_TO_{{ link.peer | upper }}_{{ link.peer_interface }}
+{%- elif link.type is arista.avd.defined('underlay_l2') -%}
+{{ link.peer | upper }}_{{ link.peer_interface }}
+{%- endif -%}
```

Comment:
I would prefer a newline at the end of the files, if it doesn't break the includes.